feat: Add gemini sample (#132)
autokitteh/samples: samples/google/gemini/README.md

---
title: Gemini

> description: Simple usage of Gemini API

Reviewer: ```suggestion
description: Simple usage of the Gemini API
```

integrations: ["gemini"]
categories: ["Samples"]
---

# Gemini Sample

This [AutoKitteh](https://github.com/autokitteh/autokitteh) project
demonstrates integration with [Gemini](https://gemini.google.com).

It sends a couple of requests to the Gemini API, and prints the responses
back to the user.

API details:

- [Gemini API](https://ai.google.dev)
- [Python client library](https://github.com/google-gemini/generative-ai-python/blob/main/docs/api/google/generativeai.md)

## How It Works

> 1. Send an HTTP GET request to trigger the workflow.
> 2. The workflow prints the responses of Gemini

Reviewer 2: ```suggestion
1. Generate content using Gemini and log the response.
2. Conduct interactive chats with Gemini and log the responses.
```

I think what you have for number 1 in "how it works" is a better fit for "Cloud Usage" section.

## Deployment & Configuration

### Cloud Usage

- Initialize your connection with Gemini through the UI

> 

Reviewer 2: I think usage should include information on how to trigger as well.
I initialized the connection, now what?
We also have manual triggers now, I think that should be included as well.
Here is the most up to date [documentation for a kittehub project](https://github.com/autokitteh/kittehub/blob/main/auth0_to_hubspot/README.md)

And I think use of [github alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) makes for more dynamic documentation that's easier to read.

> ### self-hosted

Reviewer: ```suggestion
### Self-Hosted Server
```
